_assets/js/templates/pagination/paginationButtonTemplate.js

```diff
@@ -2,9 +2,9 @@
 import {getOffsetParam, setOffsetParam} from "../../utils/offsetUtils";
 
 export default function paginationButton(offsetVal, index, resultsArr) {
-  var currentOffset = getOffsetParam();
-  var ariaLabel = "";
-  var ariaCurrent = "false";
+  const currentOffset = getOffsetParam();
+  let ariaLabel = "";
+  let ariaCurrent = "false";
```

Comment:
Not part of the PR, but should these be boolean?

Reply:
No because they need to go into the template as HTML attributes, so those are always strings. Or usually. At least in this case.